example/android/gradlew.bat

```diff
@@ -1,90 +1,90 @@
-@if "%DEBUG%" == "" @echo off
```

Comment:
we should just remove this file (and the grade.jar or some such if it's here too)

Reply:
example/android/gradle/wrapper/gradle-wrapper.jar